lib/mix/tasks/show_car_at.ex

```diff
@@ -72,6 +72,7 @@ defmodule Mix.Tasks.ShowCarAt do
       if result == car_nbr do
         {:cont, {car_nbr, dt}}
       else
+        # credo:disable-for-next-line
         IO.inspect(record, limit: :infinity)
```

Comment:
Normally I would just solve the credo complaint but this seems like a reasonable use of IO.inspect to me.